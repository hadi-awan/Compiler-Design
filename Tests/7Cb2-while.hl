// test while statement
var i;
i = 100;
while (i>=0) do
	print "i is now ";
	println i;
  i = i-7;
	end;