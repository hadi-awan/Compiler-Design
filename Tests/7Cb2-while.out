i is now 100
i is now 93
i is now 86
i is now 79
i is now 72
i is now 65
i is now 58
i is now 51
i is now 44
i is now 37
i is now 30
i is now 23
i is now 16
i is now 9
i is now 2
